### PR TITLE
Fix spark-3.0.1 shim to be released

### PR DESCRIPTION
**Pull request by revans2**
This is a follow on when the spark 3.0.1 was released and the version we are building against changed from -SNAPSHOT we didn't also update the shim so it was in a location where we will release it.

I'll file a follow on to also run integration tests against 3.0.1

### CODE REVIEW

**Comment by revans2** [comment]
build

File: docs/configs.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Name | Description | Default Value
...
 <a name="sql.format.parquet.multiThreadedRead.maxNumFilesParallel"></a>spark.rapids.sql.format.parquet.multiThreadedRead.maxNumFilesParallel|A limit on the maximum number of files per task processed in parallel on the CPU side before the file is sent to the GPU. This affects the amount of host memory used when reading the files in parallel.|2147483647
-<a name="sql.format.parquet.multiThreadedRead.numThreads"></a>spark.rapids.sql.format.parquet.multiThreadedRead.numThreads|The maximum number of threads, on the executor, to use for reading small parquet files in parallel.|20
+<a name="sql.format.parquet.multiThreadedRead.numThreads"></a>spark.rapids.sql.format.parquet.multiThreadedRead.numThreads|The maximum number of threads, on the executor, to use for reading small parquet files in parallel. This can not be changed at runtime after the executor hasstarted.|20
```
Comment (jlowe) [inline docs/configs.md:60]: Looks like there's a typo in a doc string, `hasstarted`, that got generated during this build.  Needs a followup if not fixed here.

**Review by jlowe — DISMISSED** [review]

File: docs/configs.md (hunk 1 above)
Comment (revans2) [reply docs/configs.md:60]: crud another escaped doc change.

**Comment by revans2** [comment]
build

**Comment by revans2** [comment]
@jlowe please take another look

**Review by jlowe — APPROVED** [review]

**Review by tgravescs — APPROVED** [review]
thanks, i was going to update this today as well. And thanks for fixing my typo

**Pull request merged**